src/Client.php

```diff
@@ -162,21 +162,21 @@ public function flush(?int $timeout = null): PromiseInterface
     /**
      * Assembles an event and prepares it to be sent of to Sentry.
      *
-     * @param array      $payload        the payload that will be converted to an Event
-     * @param Scope|null $scope          optional scope which enriches the Event
-     * @param bool       $withStacktrace True if the event should have and attached stacktrace
+     * @param array      $payload the payload that will be converted to an Event
+     * @param Scope|null $scope   optional scope which enriches the Event
      *
      * @return Event|null returns ready to send Event, however depending on options it can be discarded
      */
-    private function prepareEvent(array $payload, ?Scope $scope = null, bool $withStacktrace = false): ?Event
+    private function prepareEvent(array $payload, ?Scope $scope = null): ?Event
     {
         $sampleRate = $this->getOptions()->getSampleRate();
+        $withStacktrace = $this->getOptions()->shouldAttachStacktrace();
```

Comment:
Can you inline this method call in the `if` statement?

Reply:
I see you don't seem to have an issue with long lines 😆 - this will grow to 128 characters when I inline it.

Reply:
I'm more inclined to keep the code on one line, it's just a matter of preference of course and sometimes I myself find that for really long lines of code (e.g. functions with a long list of arguments) it indeed makes sense to use multiline, however most of the codebase is written this way so I prefer to stick with it as much as possible. Also our code style is implicitly based on the one of Symfony which in turn prefers one-line style :smile: By the way, the real reason I asked to inline the statement here was to avoid a variable as it's used only once

Reply:
yeah - the reason I used the variable was to keep the line short.

in the end it's totally a matter of taste and I gladly adopt to the existing style.

Reply:
fixed